examples/thermostat/thermostat-common/thermostat.zap

```diff
@@ -9683,6 +9683,66 @@
               "maxInterval": 65344,
               "reportableChange": 0
             },
+            {
+              "name": "abs min heat setpoint limit",
+              "code": 3,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 0,
+              "bounded": 0,
+              "defaultValue": "1000",
```

Comment:
Where is this "1000" coming from?  That does not match the value in the XML or the spec (both of which are "700").

Reply:
These were values I used  as test values to validate the functionality. I will change them back to their default values in all 8 cases.

Reply:
Changing all 8 attribute values in zap to decimal defaults per spec.